.github/workflows/trivy-scan.yml

```
@@ -64,7 +64,7 @@ jobs:
       - name: Set up Go 1.x
         uses: actions/setup-go@v1
```

```suggestion
        uses: actions/setup-go@v3
```
Let us maybe use this opportunity to align the version of the action with